common/vts.proto

```diff
@@ -15,6 +19,122 @@ message CloseArgs {
 }
 
 message CloseResponse {
+    Status status = 1;
+}
+
+message OpenArgs {
+}
+
+message OpenResponse {
+    Status status = 1;
+}
+
+enum TaType {
+    TA_UNSPECIFIED = 0;
+    TA_RAWPUBLICKEY = 1;
+    // Refer https://github.com/veraison/veraison/issues/76, to introduce certs here!
+    TA_INVALID = 255;
+}
+
+message EndorsementID {
```

Comment:
Shouldn't this be URI as defined here; https://github.com/veraison/veraison/blob/main/kvstore/README.md?

Reply:
This is bound to go in https://github.com/veraison/veraison/issues/122